events/interaction/features/sessions/handle-clock-out-button.js

```diff
@@ -0,0 +1,34 @@
+import { MessageFlags } from "discord.js";
+import { buildClockOutMessagePayload } from "../../../../features/session/messageBuilder.js";
+import { cancelTimer } from "../../../../features/session/timerManager.js";
```

Comment:
This file and function doesn't exist, did you forget to commit it? or commit it somewhere else? 🤪